taskrun/ClusterTask.py

```diff
@@ -224,6 +243,8 @@ def _build_command(self):
       else:
         cmd.extend(['-e', os.devnull])
       cmd.append(self._command)
+      if self._log_file:
+        cmd.extend(['2>', self._log_file])
```

Comment:
Take a look at ProcessTask. You want to use the stdout and stderr of the subprocess, not console redirects. If you use stdout=<file> and stderr=PIPE, then both streams will be directly to the file you specify.